### PR TITLE
updates with azure monitor extension onboarding with GKE

### PR DESCRIPTION
**Pull request by sowsan**

### CODE REVIEW

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/arc_extensions/azure_monitor/index.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    "password": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "tenant": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+    }
+    ```
+
+    > **Note: It is optional but highly recommended to scope the SP to a specific [Azure subscription and resource group](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest)**
+
+## Create Azure monitor extensions instance
+
+To create a new extension Instance, use `k8s-extension create`, passing in values for the mandatory parameters. The below example creates an instance of Azure Monitor extension on your Arc enabled Kubernetes cluster. An auotmation script file is provided part of jumpstart.
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/day2/arc_extensions/azure_monitor/index.md:64]:
```suggestion
To create a new extension Instance, we will use the _k8s-extension create_ command while passing in values for the mandatory parameters. This scenario provides you with the automation to deploy the Azure Monitor extension on your Azure Arc enabled Kubernetes cluster.
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/arc_extensions/azure_monitor/index.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+    ![Screenshot showing uploading the script file to cloud shell](./07.png)
+
+    The script will:
+
+  * Login to your Azure subscription using the SPN credentials
+  * Add or Update the connectedk8s and k8s-extension
+  * Create monitor k8s extension instance
+
+* Now if you can see that the monitoring is enabled once you visit Container Insights section of Azure Arc enabled kubernetes cluster in Azure.
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/day2/arc_extensions/azure_monitor/index.md:90]:
```suggestion
* Now if you can see that the monitoring is enabled once you visit the Container Insights section of the Azure Arc enabled Kubernetes cluster resource in Azure.
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/arc_extensions/azure_monitor/index.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    ![Screenshot showing GKE cluster in GCP console](./03.png)
+
+    ![Screenshot showing connection to GKE cluster in GCP console](./04.png)
+
+    ![Screenshot showing uploading the script file to cloud shell](./07.png)
+
+    The script will:
+
+  * Login to your Azure subscription using the SPN credentials
+  * Add or Update the connectedk8s and k8s-extension
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/day2/arc_extensions/azure_monitor/index.md:87]:
```suggestion
  * Add or Update your local _connectedk8s_ and _k8s-extension_ Azure CLI extensions
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/arc_extensions/azure_monitor/index.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+### Delete extension instance
+
+The following command only deletes the extension instance, but doesn't delete the Log Analytics workspace. The data within the Log Analytics resource is left intact. You can visit the Insights seciton of Azure Arc resource in Azure Portal and can see the onboarding state as not connected.
+
+```bash
+az k8s-extension delete --name azuremonitor-containers --cluster-type connectedClusters --cluster-name <cluster-name> --resource-group <resource-group>
+```
+
+ ![Screenshot showing Azure Portal with Azure Arc enabled Kubernetes resource Insights](./02.png)
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/day2/arc_extensions/azure_monitor/index.md:106]:
```suggestion
 ![Screenshot showing Azure Portal with Azure Arc enabled Kubernetes resource Insights](./02.png)
```
Change filename to _07.png_

See [this section](https://azurearcjumpstart.io/scenario_guidelines/#screenshots) in the write-up guildeines.

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/arc_extensions/azure_monitor/index.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+* Edit the environment variables in the script to match your environment parameters and upload it to the Cloud Shell environment. After than run it using the ```. ./azure_monitor_k8s_extension.sh``` command.
+
+    > **Note: The extra dot is due to the shell script has an *export* function and needs to have the vars exported in the same shell session as the rest of the commands.**
+
+    ![Screenshot showing GKE cluster in GCP console](./03.png)
+
+    ![Screenshot showing connection to GKE cluster in GCP console](./04.png)
+
+    ![Screenshot showing uploading the script file to cloud shell](./07.png)
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/day2/arc_extensions/azure_monitor/index.md:82]: img jump 04<>07

See [this section](https://azurearcjumpstart.io/scenario_guidelines/#screenshots) in the write-up guildeines.

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/arc_extensions/index.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+---
+type: docs
+title: "Kubernetes Extensions"
+linkTitle: "Kubernetes Extensions"
+weight: 6
+description: >-
+  Kubernetes extensions are add-ons for Kubernetes clusters. The extensions feature on Azure Arc enabled Kubernetes clusters enables usage of Azure Resource Manager based APIs, CLI and portal UX for deployment of extension components (Helm charts in initial release) and will also provide lifecycle management capabilities such as auto/manual extension version upgrades for the extensions. The scenarios in this section will guide on how to use Kubernetes extensions in your Azure Arc enabled Kubernetes cluster in an automated fashion.
+---
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/day2/arc_extensions/index.md:8]: This should be incorporated under the [Unified Operations Use Cases](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/day2/) section and not in its own section. 

I recommend updating the index file for that section.

**Review by likamrat — COMMENTED** [review]
- Change docs folder and links to _docs\azure_arc_jumpstart\azure_arc_k8s\day2\gke\gke_monitor_extension_
- Change script folder to _azure_arc_k8s_jumpstart\gke\gke_monitor_extension_
- Some change img filename extension to _png_ (Some are PNG) and update links
- Fix img links "jumps"
- img01 - hide subscription id
- img07 - Reduce the size of the step number. Also, might better to use the _ls -l_ command and just have the step number next to the filename
- Change section description positioning
- Minor lingo updates

**Review by lanicolas — COMMENTED** [review]
@sowsan index.md files should start with "_" so they should be _index.md

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/arc_extensions/azure_monitor/index.md (hunk 1 above)
Comment (sowsan) [reply docs/azure_arc_jumpstart/azure_arc_k8s/day2/arc_extensions/azure_monitor/index.md]: fixed and let me know if there any further updates needed. Thanks!

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/arc_extensions/azure_monitor/index.md (hunk 2 above)
Comment (sowsan) [reply docs/azure_arc_jumpstart/azure_arc_k8s/day2/arc_extensions/azure_monitor/index.md]: fixed and let me know if there any further updates needed. Thanks!

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/arc_extensions/azure_monitor/index.md (hunk 3 above)
Comment (sowsan) [reply docs/azure_arc_jumpstart/azure_arc_k8s/day2/arc_extensions/azure_monitor/index.md]: fixed and let me know if there any further updates needed. Thanks!

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/arc_extensions/azure_monitor/index.md (hunk 4 above)
Comment (sowsan) [reply docs/azure_arc_jumpstart/azure_arc_k8s/day2/arc_extensions/azure_monitor/index.md]: fixed and let me know if there any further updates needed. Thanks!

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/arc_extensions/azure_monitor/index.md (hunk 5 above)
Comment (sowsan) [reply docs/azure_arc_jumpstart/azure_arc_k8s/day2/arc_extensions/azure_monitor/index.md]: fixed and let me know if there any further updates needed. Thanks!

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/arc_extensions/index.md (hunk 6 above)
Comment (sowsan) [reply docs/azure_arc_jumpstart/azure_arc_k8s/day2/arc_extensions/index.md]: fixed and let me know if there any further updates needed. Thanks!

**Review by likamrat — APPROVED** [review]
LGTM. Great work on this!

**Pull request merged**